packages/lib-react-components/src/CloseButton/README.md

```diff
@@ -0,0 +1,8 @@
+# CloseButton
+
+A styled Grommet `Button` component that displays a close icon. Has the same API as Grommet's `Button`.
+
+## props
+
+- `closeFn` (function): the function called onClick of the button
+- `className` (string): a string class name
```

Comment:
You shouldn't need `className`, it should be passed down by the base Button component: [link](https://github.com/grommet/grommet/blob/master/src/js/components/Button/Button.js#L121). Have you tried not explicitly passing it in?

Reply:
(it certainly shouldn't be needed by default, but willing to be proved wrong)

Reply:
I suppose it doesn't have to be explicitly declared.